packages/knip/src/plugins/angular/update-types.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+set -eu
+angular_plugin_dir="$(cd "$(dirname "$0")" && pwd)"
+tmp_dir="$(mktemp -d)"
+cleanup() {
+  rm  -rf "$tmp_dir"
+}
+trap cleanup EXIT
+git clone --depth 1 https://github.com/angular/angular-cli "$tmp_dir"
+cd "$tmp_dir/packages/angular/cli/lib/config"
+bunx json-schema-to-typescript -i workspace-schema.json -o "$angular_plugin_dir/types.ts";
```

Comment:
Knip reported unused exports. We have a few options here:

- Add as entry file (meh)
- Ignore in root `knip.json` (fine, but doesn't look super great in knip repo itself)
- Fix up unused exports using knip:

```suggestion
bunx json-schema-to-typescript -i workspace-schema.json -o "$angular_plugin_dir/types.ts";
$root_dir/packages/knip/bin/knip-bun.js --directory "$root_dir" --fix
```